### ISSUE TITLE
Error: stdout maxBuffer exceeded

### ISSUE THREAD

**Issue by degendra** [jmervine/node-yslowjs#5]
Thank you for this library. When i run it on http://www.dailymail.co.uk/home/index.html I get following error
`Error: stdout maxBuffer exceeded
    at Socket.<anonymous> (child_process.js:266:14)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:146:16)
    at Socket.Readable.push (_stream_readable.js:110:10)
    at Pipe.onread (net.js:523:20)`

When i change 
`this.phantom = new Phapper(this.script, args);`
to
`this.phantom = new Phapper(this.script, args, {maxBuffer: 1024*1024});`
in file lib/yslow.js then it works. Please provide a way to set maxBuffer size. 
Thank you.

**Comment by jmervine** [comment]
Published 0.3.3 which supports...

``` javascript
var phantom_opts = { maxBuffer: 1024*1024 };
var ys = new YSlow('http://www.example.com', [], { phantomjs: phantom_opts });
```

That should do the trick.

**Comment by degendra** [comment]
Thank you.
